src/pages/Episode/Episode.js

```diff
@@ -1,26 +1,105 @@
 import React, { Component } from "react";
 
+import axios from "axios";
 import Layout from "../../components/Layout";
-// import CharacterCard from "../../components/CharacterCard";
+import CharacterCard from "../../components/CharacterCard";
+
+import { getEpisode } from "../../api";
 
 class Episode extends Component {
   constructor(props) {
     super(props);
 
-    this.state = {};
-    // episode: null,
-    // characters: [],
-    // hasLoaded: false,
-    // hasError: false,
-    // errorMessage: null,
+    this.state = {
+      episode: [],
+      characters: [],
+      hasLoaded: false,
+      hasError: false,
+      errorMessage: null,
+    };
+    this.loadEpisode = this.loadEpisode.bind(this);
+  }
+
+  componentDidMount() {
+    const { match } = this.props;
+    const { episodeId } = match.params;
+
+    this.loadEpisode(episodeId);
+  }
+
+  async loadEpisode(episodeId) {
+    try {
+      const { data } = await getEpisode(episodeId);
+
+      // eslint-disable-next-line
+      console.log(data);
```

Comment:
We should not leave any console logs in finished projects. We can always comment them if we think they can be used for later on